modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/MeResourceManagerTest.java

```diff
@@ -0,0 +1,1553 @@
+            "\"meta\": {\n" +
+            "    \"created\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"location\": \"ENDPOINT/008bba85-451d-414b-87de-c03b5a1f4217\",\n" +
+            "    \"lastModified\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"resourceType\": \"User\"\n" +
+            "},\n" +
+            "  \"Operations\": [\n" +
+            "    {\n" +
+            "      \"op\": \"replace\",\n" + //REPLACE Operation
+            "      \"value\": {\n" +
+            "        \"nickName\": \"shaggy\"\n" +
+            "      }\n" +
+            "    }\n" +
+            "  ],   \n" +
+            " \"name\": {\n" +
+            "    \"givenName\": \"Kim\",\n" +
+            "    \"familyName\": \"Berry\",\n" +
+            "    \"formatted\": \"Kim Berry\"\n" +
+            "  },\n" +
+            "  \"userName\": \"kimjohn  \",\n" +
+            "  \"password\": \"kim123\",\n" +
+            "  \"id\": \"123\",\n" +
+            "  \"emails\": [\n" +
+            "      {\n" +
+            "        \"type\": \"home\",\n" +
+            "        \"value\": \"john@gmail.com\",\n" +
+            "         \"primary\": true\n" +
+            "      }\n" +
+            "  ]\n" +
+            "}";
+
+    private final String newUserSCIMObjectStringForPatchWithReplaceUpdated = "{\n" +
+            "  \"schemas\": \n" +
+            "    [\"urn:ietf:params:scim:schemas:core:2.0:User\",\n" +
+            "    \"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User\",\n" +
+            "    \"urn:ietf:params:scim:api:messages:2.0:PatchOp\"\n" +
+            "    ],\n" +
+            "\"meta\": {\n" +
+            "    \"created\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"location\": \"ENDPOINT/008bba85-451d-414b-87de-c03b5a1f4217\",\n" +
+            "    \"lastModified\": \"2018-08-17T10:34:29Z\",\n" +
+            "    \"resourceType\": \"User\"\n" +
+            "},\n" +
+            "  \"Operations\": [\n" +
+            "    {\n" +
+            "      \"op\": \"replace\",\n" + //REPLACE Operation
+            "      \"value\": {\n" +
+            "        \"nickName\": \"shaggy\"\n" +
+            "      }\n" +
+            "    }\n" +
+            "  ],   \n" +
+            " \"name\": {\n" +
+            "    \"givenName\": \"Kim\",\n" +
+            "    \"familyName\": \"Berry\",\n" +
+            "    \"formatted\": \"Kim Berry\"\n" +
+            "  },\n" +
+            "  \"userName\": \"kimjohn  \",\n" +
+            "  \"password\": \"kim123\",\n" +
+            "  \"id\": \"123\",\n" +
+            "  \"emails\": [\n" +
+            "      {\n" +
+            "        \"type\": \"home\",\n" +
+            "        \"value\": \"johnnew@gmail.com\",\n" +
+            "         \"primary\": true\n" +
+            "      }\n" +
+            "  ]\n" +
+            "}";
+
+    private final String endpoint = "https://localhost:9443/scim2/Me";
+
+    private final int internalError = 500;
```

Comment:
These constants are used in ResponseCodeConstants. Let's use them instead of defining again.

Reply:
Fixed in commit d8f6f1c18502d1174c97a568b559f353a6b945a6